### ISSUE TITLE
Release extension without push notifications

### ISSUE THREAD

**Issue by matejdro** [joaomgcd/JoinFirefox#6]
Since push notifications appear to be problematic, here is a crazy idea: What if you release the extension without push ability? Instead, provide manual refresh button. It is not nearly as elegant as push, but for me it would fulfill my two major use cases for Join:

1. Whenever my phone rings or chimes, I would just press on refresh button and I would see the notification / who is calling me

2. Whenever I push files to the browser, I would press refresh button and file would start downloading.

**Comment by dthigpen** [comment]
That works for most of my use cases as well, so I'd support it. When (or if) Firefox decides to open push notifications to Add-ons it would be easier to integrate the full Join service.

**Comment by tehniemer** [comment]
I'll third this idea.

**Comment by Aephir** [comment]
Same here. After migrating to Firefox and Opera Neon for browsing, I mostly miss the ability to quickly write texts from the add-on (without having to open the web version).

My phone is usually nearby, so it's not like I'm missing the "push" (well, it's not the main appeal of the Join extension), it's just nice to not break your workflow too much by having to pick up the phone to see/answer a text.

**Comment by Rohaq** [comment]
Not to mention that if you can make the request manually, you can probably automate it on a timer.

I'd settle for waiting a little while before they came through, if it can't be done on a push.

**Comment by GlydetheSurfer** [comment]
I also agree.. i really want the join extension on firefox since its the only thing stopping me from switching from chrome...
